assets/src/helpers.js

```diff
@@ -498,3 +498,21 @@ export const addBackgroundColorToOverlay = ( overlayStyle, backgroundColors ) =>
 	}
 	return overlayStyle;
 };
+
+/**
+ * Converts hex to rgba.
+ *
+ * @param {string} hex Hex value.
+ * @param {number} opacity Opacity.
+ * @return {string|undefined} Rgba value.
+ */
+export const getRgbaFromHex = ( hex, opacity ) => {
+	if ( ! hex ) {
+		return undefined;
+	}
+	hex = hex.replace( '#', '' );
+	const r = parseInt( hex.substring( 0, 2 ), 16 );
+	const g = parseInt( hex.substring( 2, 4 ), 16 );
+	const b = parseInt( hex.substring( 4, 6 ), 16 );
+	return `rgba( ${ r }, ${ g }, ${ b }, ${ opacity / 100 })`;
```

Comment:
If this just returns `[ r, g, b, opacity / 100 ]`, it would be more re-usable and also allow conversion back to hex. In the block we can then just use `const [ r, g, b, a ] = getRgbaFromHex( ... )` and `rgba( ${ r }, ${ g }, ${ b }, ${ a })\``. Alternatively, I'd probably rename this from `getRgbaFromHex` to `rgbaCssValueFromHex` or something.

Reply:
Sure, makes sense, I'll change it to return `[ r, g, b, opacity / 100 ]` instead.